low-level/framework/utils/utility.py

```diff
@@ -166,6 +167,48 @@ def get_current_time():
         """Returns the time as integer number in seconds since the epoch in UTC."""
         return int(time.time())
 
+    @staticmethod
+    def validate_storage_type_support(log, Error, storage_type):
+        """Check for supported storage type."""
+        logger.debug(log.svc_log(f"Storage Type:{storage_type}"))
+        if not storage_type:
+            msg = "ConfigError: storage type is unknown."
+            logger.error(log.svc_log(msg))
+            raise Error(errno.EINVAL, msg)
+        if storage_type.lower() not in const.RESOURCE_MAP["storage_type_supported"]:
+            msg = f"{log.service} provider is not supported for storage type '{storage_type}'"
+            logger.error(log.svc_log(msg))
```

Comment:
All the utils code should be in the utils repo. If its really utils.

Reply:
Yes. moved this code to the appropriate type file